### ISSUE TITLE
disable_functions

### ISSUE THREAD

**Issue by henrywood** [jumbaeric/PhpInfoGuard#1]
Isn't this whole project basically the same as this oneliner (in a php ini file):

disable_functions = "phpinfo"

?

**Comment by jumbaeric** [comment]
Not really, the difference is the one liner on php ini completely disables phpinfo function on the web server, this project keeps the phpinfo function enabled on the web server while limiting access to only authorised users.
